packages/backend-defaults/src/entrypoints/rootHttpRouter/http/MiddlewareFactory.ts

```diff
@@ -45,6 +46,29 @@ import {
 import { NotImplementedError } from '@backstage/errors';
 import { applyInternalErrorFilter } from './applyInternalErrorFilter';
 
+const getLogMessage = morgan.compile(
+  '[:date[clf]] ":method :url HTTP/:http-version" :status :res[content-length] ":referrer" ":user-agent"',
+);
+
+function getLogMeta(
+  tokens: TokenIndexer,
+  req: IncomingMessage,
+  res: ServerResponse,
+) {
+  const status = Number(tokens.status(req, res));
+  const contentLength = Number(tokens.res(req, res, 'content-length'));
+  return {
+    date: tokens.date(req, res, 'iso'),
```

Comment:
Should we skip the date perhaps? or is it useful somehow even if we already have the log timestamp?

Reply:
Actually, I realized that this basically does half the work of formatting the message already, I think it's worth expanding these `tokens.*` calls, construct the message ourselves, and get rid of `morgan` altogether.

Reply:
I'll give it a try

Reply:
I removed morgan usage, ptal